mmv1/third_party/terraform/services/dataplex/resource_dataplex_entry_meta.yaml

```diff
@@ -1,36 +0,0 @@
-resource: 'google_dataplex_entry'
```

Comment:
this file isn't necessary because we already automatically generate a meta file for this resource: https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/services/dataplex/resource_dataplex_entry_generated_meta.yaml